### ISSUE TITLE
Keyboard shortcuts for left-right

### ISSUE THREAD

**Issue by e-stonia** [peterdotjs/tab-resize#60]
Hey, love the extension! Just keyboard shortcuts for left-right would be great. Ctrl+Shift+2 does a great job but I have to choose from menu if I want to change that to the other side. Would be super fast to do Ctl+Shift+3 for example in another window and have it in the other side.

**Comment by jrownak** [comment]
YES! This would be AWESOME! e-stonia, I'm with you. I currently use Ctrl+Shift+0 to activate the extension so I can click right when I need it, but it would be awesome to have the shortcut to just move it over. Also, I use Ctrl+Shift+ 1,2,3,4 for various sizes (smaller to larger as well.
